lib/rules/accessor-pairs.js

```diff
@@ -63,3 +63,11 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.documentation = {
+    errors: ["Getter is not present", "Setter is not present"],
```

Comment:
Second one would probably be a bit better, but it would break all of the plugins. I would rather go for backwards compatible one. So first one in this case.